### PR TITLE
Join tables

### PR DESCRIPTION
**Pull request by touredri**
In this PR we used the primary and foreign keys and then joined the table, below the queries' screenshot.

Capture d’écran 2023-06-08 223753
Capture d’écran 2023-06-08 225657
Capture d’écran 2023-06-08 230239
Capture d’écran 2023-06-09 002111

Capture d’écran 2023-06-09 011359
Capture d’écran 2023-06-09 012910
Capture d’écran 2023-06-09 014016

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -30,3 +30,22 @@ select count(*) from animals;
...
+select species_id, count(*) as count from animals group by species_id;
+select name from animals inner join species on animals.species_id = species.id where species_id = owners_id;
+select name from animals inner join owners  on animals.owners_id = owners.id  where animals.escape_attempts = 0;
+
+SELECT o.full_name, COUNT(*) AS count
+FROM owners o
+JOIN animals a ON o.id = a.owner_id
+GROUP BY o.full_name
+ORDER BY count DESC
+LIMIT 1;
```
Comment (alexander16108) [inline queries.sql]: please input tthe following queries into the queries.sql file

``
   SELECT animals.* FROM animals
   LEFT JOIN owners
   ON animals.owner_id = owners.id
   LEFT JOIN species
   ON animals.species_id = species.id
   WHERE owners.full_name = 'Jennifer Orwell'
   AND species.name = 'Digimon';
``

please add the screenshot of the following queries

- List of all animals that are Pokemon (their type is Pokemon).
- List all owners and their animals, remember to include those that don't own any animal.
- List all Digimon owned by Jennifer Orwell.

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @touredri , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

- please fix the broken link to your mit license it is unprofessional to have broken link in your code base
- please remove the live version link from your readme file as there currently no live version for this project and replace with a boilerplate text or remove completely

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (touredri) [reply queries.sql]: OK

File: queries.sql [hunk 2]
```diff
@@ -30,3 +30,29 @@ select count(*) from animals;
...
+
+SELECT animals.* FROM animals 
+LEFT JOIN owners ON animals.owner_id = owners.id 
+LEFT JOIN species ON animals.species_id = species.id 
+WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
+
```
Comment (brainconnect93) [inline queries.sql:57]: - [ ] As suggested bybthe previous reviewer I would kindly suggest that you should please add the screenshot of the following queries to your Pull Request 

- List of all animals that are Pokemon (their type is Pokemon).
- List all owners and their animals, remember to include those that don't own any animal.
- List all Digimon owned by Jennifer Orwell.

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @touredri, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Good Points 👍

- Has a proper Pull Request. ✔️
- Query Multiple Tables milestone done perfectly ✔️
- Add a Screenshot of all queries to the PR ✔️
- Good use of Git flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (touredri) [reply queries.sql:57]: @brainconnect93 kindly check the PR description because I added it.

Comment (touredri) [reply queries.sql:57]: I forgot to update it , Thanks @brainconnect93

Comment (rloterh) [reply queries.sql:57]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Drissa,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job implementing recommended changes!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Kindly ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe. 
- The FAQ link in the table of contents has no corresponding content in the ReadMe, let's therefore ensure to remove it.

**Let's remove/update the underlined aspects in the ReadMe:**

image

image


Let's improve upon this aspect to make your ReadMe/ repository stand out even better! 😉

Cheers..Happy coding!💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**